### PR TITLE
CGPROD 2770 Collections

### PR DESCRIPTION
**Pull request by FostUK**
- Adds the concept of collections which link to catalogues.
- Changed catalogue loader to initialise collections instead.

## Future Considerations
* Do we need to expose the catalogue on collections ever?
* `itemList.set` might need a deeper merge.
* Do we need namespacing these under the genie localstorage tag?

### CODE REVIEW

File: themes/default/items/characters.json5 [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+        title: "Bobson Dugnut",
+        description: "The greatest baseball player to ever live."
+    },
+    {
+        id: "sleveMcDichael",    //must be unique - kebab case it? If we stick to alphanumerics it can be pulled out directly.
+        title: "Sleve McDichael",
+        description: "He could have been the greatest if it wasn't for his bad luck with injuries."
+    },
+
+]
```
Comment (AdamXA) [inline themes/default/items/characters.json5:13]: can we add @AnthonyGoodchild  to the character list

File: themes/default/items/game-items.json5 [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        description: "Basic wooden crossbow with standard range.",
+        tags: ["weapon"],
+    },
+    {
+        id: "bread",    //must be unique - kebab case it? If we stick to alphanumerics it can be pulled out directly.
+        title: "Bread",
+        description: "A loaf of bread.",
+        tags: ["food"],
+    },
+]
```
Comment (AdamXA) [inline themes/default/items/game-items.json5:23]: no frying pan weapon?

**Review by AdamXA — APPROVED** [review]
4/5 - characters and items let you down on this one

File: themes/default/items/characters.json5 (hunk 1 above)
Comment (AnthonyGoodchild) [reply themes/default/items/characters.json5:13]: Liron Man 🦁

File: themes/default/items/game-items.json5 (hunk 2 above)
Comment (FostUK) [reply themes/default/items/game-items.json5:23]: `sfx: "files/sounds/PaaaAAAAAAANNNG.mp4"` ?

**Pull request merged**